apps/loan/src/components/PageLlamaMarkets/LendingMarketsTable.tsx

```diff
@@ -41,37 +52,61 @@ const columns = [
     meta: { type: 'numeric' },
     size: ColumnWidth.sm,
   }),
+  // hidden columns
+  columnHelper.accessor('blockchainId', {
+    filterFn: (row, columnId, filterValue) => !filterValue?.length || filterValue.includes(row.getValue(columnId)),
+    meta: { hidden: true },
+  }),
 ] satisfies ColumnDef<LendingVault, any>[]
 
+const DEFAULT_SORT = [{ id: 'totalSupplied.usdTotal', desc: true }]
+
 export const LendingMarketsTable = ({
   onReload,
-  data,
+  data: sourceData,
   headerHeight,
 }: {
   onReload: () => void
   data: LendingVault[]
   headerHeight: string
-}) => (
-  <Stack
-    sx={{
-      marginBlockStart: Spacing.xl,
-      marginBlockEnd: Spacing.xxl,
-      maxWidth: MaxWidth.table,
-      minWidth: MinWidth.table,
-      backgroundColor: (t) => t.design.Layer[1].Fill,
-    }}
-  >
-    <TableFilters
-      title={t`Llamalend Markets`}
-      subtitle={t`Select a market to view more details`}
-      onReload={onReload}
-      learnMoreUrl="https://docs.curve.fi/lending/overview/"
-    />
-    <DataTable
-      data={useMemo(() => data.filter((d) => d.usdTotal > 0).sort((a, b) => b.usdTotal - a.usdTotal), [data])}
-      defaultSort={[{ id: 'totalSupplied.usdTotal', desc: true }]}
-      columns={columns}
-      headerHeight={headerHeight}
-    />
-  </Stack>
-)
+}) => {
+  const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([])
+  const data = useMemo(
+    () => sourceData.filter((d) => d.usdTotal > 0).sort((a, b) => b.usdTotal - a.usdTotal),
```

Comment:
Is this sorting actually needed when we have a default sorting? Alternatively you could also bake the `usdTotal > 0` filtering into Tanstack column filtering, but I don't have a strong opinion on that; do whatever you want.